portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,31 @@
 #content {
   margin-left: auto;
   margin-right: auto;
+  color: white;
   width: 650px;
 }
 
+#header {
+  margin-left: auto;
```

Comment:
You can shortcut any CSS property that applies to all sides of a "box"

`margin: <value>;` will apply value to all sides.
`margin: <value1> <value2>;` will apply value1 to all top and bottom and value 2 to left and right
`margin: <value1> <value2> <value3> <value4>;` will apply value1 to top, value2 to right, value3 to bottom, and value4 to top.

It's always clockwise from top. There's a variant that uses 3 values but I recommend against using that one.